lib/commands/bugs.js

```diff
@@ -22,7 +22,10 @@ class Bugs extends PackageUrlCmd {
     // try to get it from the repo, if possible
     const info = this.hostedFromMani(mani)
     if (info) {
```

Comment:
It's been quite a while since this file had eyes on it, feel free to clean this up to use something like `const infoUrl = info?.bugs()` to remove an if statement and use some newer syntax.  The `engines` declaration of npm allows for that syntax to be used.

You don't have to but it'd be nice to polish this while we're here.